agent/strategy/quality/techdebt/pmdcpd.py

```diff
@@ -0,0 +1,27 @@
+from agent.utils import run_logged
+import os
+
+class PmdCpd:
+    def run(self, directories, techdebt_dir):
+
+        languages = ["cpp","cs","ecmascript","java","objectivec","php","ruby", "scala"]
+        pmdcpdparser_output_folder_path = os.path.join(techdebt_dir,'PMDCPDParser')
+        if not os.path.exists(pmdcpdparser_output_folder_path):
+            os.makedirs(pmdcpdparser_output_folder_path)
```

Comment:
`techdebt_dir` is a Path object from [Pathlib](https://docs.python.org/3/library/pathlib.html#basic-use) -- so it can be joined with other path segments with the `/` operator, and we can use the `mkdir` method on it directly.

Additionally the `import os` can then be removed.

```suggestion
        pmdcpdparser_output_folder_path = techdebt_dir / "pmdcpd"
        pmdcpdparser_output_folder_path.mkdir(exist_ok=True)

```